project/Build.scala

```diff
@@ -920,6 +922,13 @@ object Build {
         IO.write(baseDirectory.value / "sbt-dotty-sbt", pluginText)
         IO.write(baseDirectory.value / "dotty-bootstrapped.version", dottyVersion)
       },
+      updateCommunityBuild := testOnly.in(Test).toTask(
+        " " + "dotty.communitybuild.CommunityBuildUpdate" +
+        " " + "-- --inclue-categories=dotty.communitybuild.UpdateCategory").value,
```

Comment:
typo ? inclue -> include (same below)

Reply:
Also I'd just put all of this in one String, makes it easier to copy-paste into sbt